trees/app/components/json-pretty.js

```diff
@@ -20,12 +20,12 @@ var JsonPrettyComponent = Ember.Component.extend({
             out = this.highlightSyntax(out);
         }
         return new Ember.String.htmlSafe(out);
-    }.property('obj'),
+    }),
 
     //Thanks to: http://jsfiddle.net/KJQ9K/
     highlightSyntax: function(json) {
         json = json.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
-        return json.replace(/("(\\u[a-zA-Z0-9]{4}|\\[^u]|[^\\"])*"(\s*:)?|\b(true|false|null)\b|-?\d+(?:\.\d*)?(?:[eE][+\-]?\d+)?)/g, function (match) {
+        return json.replace(/("(\\u[a-zA-Z0-9]{4}|\\[^u]|[^\\"])*"(\s*:)?|\b(true|false|null)\b|-?\d+(?:\.\d*)?(?:[eE][+\-]?\d+)?)/g, function(match) {
```

Comment:
Minor: This diff line seems extraneous, looks like it just removes a space character. Please revert this change.

Reply:
Good catch. My default linting config creeping in :)